Documentation/helm-operator.md

```diff
@@ -114,7 +114,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `mon.healthCheckInterval` | The frequency for the operator to check the mon health          | `45s`                                                  |
 | `mon.monOutTimeout`       | The time to wait before failing over an unhealthy mon           | `300s`                                                 |
 
-&ast; For Kubernetes 1.9.x `agent.flexVolumeDirPath` should be changed to `/var/lib/kubelet/volumeplugins/`. [Flexvolume documentation](flexvolume.md#for-kubernetes--19x)
+&ast; For Kubernetes 1.9.x `agent.flexVolumeDirPath` should be changed to `/var/lib/kubelet/volumeplugins/`. For more information, please referrer to the [Rook flexvolume documentation](flexvolume.md)
```

Comment:
the word should be "refer", not "referrer".

Reply:
Both are English words though, right? :D